packages/react/src/forms/FormControl/FormControl.module.css

```diff
@@ -60,6 +60,10 @@
   width: 100%;
 }
 
+:global(.ControlGroup) .FormControl-label {
```

Comment:
I don't think this will work as expected in the compiled output, as .ControlGroup will be hashed in that components stylesheet but not here. Is there another way we do this from inside the ControlGroup component with higher specificity selector instead?

Reply:
Resolved as per our conversation